README.md

```diff
@@ -1 +1,114 @@
-# boilerplate-for-ICO-Staking-new-
\ No newline at end of file
+# Boilerplate for ICO & Staking
+
+This boilerplate is for ICO & Staking application.
+Using this boilerplate, you can create ICO & Staking website.
+
+![image](imgs/img.png)
+
+![image](imgs/img_19.png)
+
+# How to create this application?
+
+## Deploy smart contracts
+
+### [Deploy ICO contract](https://app.bunzz.dev/module-templates/f5038e83-08c0-415e-953d-72a1dd07d111)
+
+Please deploy ICO smart contract
+
+![img_2.png](imgs/img_2.png)
+
+![img_3.png](imgs/img_3.png)
+
+![img_4.png](imgs/img_4.png)
+
+If you have already an ERC20 token for ICO, you can skip this.
+
+![img_5.png](imgs/img_5.png)
+
+![img_7.png](imgs/img_7.png)
+
+![img_8.png](imgs/img_8.png)
+
+![img_9.png](imgs/img_9.png)
+
+![img_11.png](imgs/img_11.png)
+
+![img_12.png](imgs/img_12.png)
```

Comment:
Please add a description for each screenshot and follow the action👍
Also, please describe what value for arguments you passed on the generator page🙏